backend/src/election/mod.rs

```diff
@@ -90,3 +91,43 @@ pub async fn election_status(
     let statuses = polling_station_repo.status(id).await?;
     Ok(Json(ElectionStatusResponse { statuses }))
 }
+
+/// Download a generated PDF
+#[utoipa::path(
+    get,
+    path = "/api/elections/{election_id}/download_pdf",
+    responses(
+        (
+            status = 200,
+            description = "PDF",
+            content_type="application/pdf",
+            headers(
+                ("Content-Type", description = "application/pdf; charset=utf-8"),
```

Comment:
There's no charset because PDF is a binary format:
```suggestion
                ("Content-Type", description = "application/pdf"),
```

Reply:
Sorry, I'm doing these corrections too hasty. Fixed in [f643473](https://github.com/kiesraad/abacus/commit/f643473ac5a3411692c4acd3091eafab0ded1458)